src/js/core.js

```diff
@@ -130,6 +130,21 @@
             node.parentElement.removeChild(node);
 
             event.preventDefault();
+        } else if (MediumEditor.util.isKey(event, MediumEditor.util.keyCode.BACKSPACE) &&
+                (tagName === 'blockquote' || node.parentElement.nodeName.toLowerCase() === 'blockquote') &&
+                MediumEditor.selection.getCaretOffsets(node).left === 0) {
+
+            // when cursor is at the begining of the element and the element is <blockquote>
+            // then pressing backspace key should change the <blockquote> to a <p> tag
+            event.preventDefault();
+
+            //if <blockquote> is the parent element, then make it the current node
+            if (node.parentElement.nodeName.toLowerCase() === 'blockquote') {
```

Comment:
the conditional is added if ```blockquote``` has child elements inside it. For ex: ```<blockquote><b>lorem</b></blockquote>```. So need to select the whole ```blockquote``` element and then call ```execFormatBlock``` on it.

Thoughts?